tools/data/textrecog/funsd_converter.py

```diff
@@ -166,8 +168,17 @@ def generate_ann(root_path, split, image_infos, preserve_vertical):
             index += 1
             dst_img_path = osp.join(dst_image_root, dst_img_name)
             mmcv.imwrite(dst_img, dst_img_path)
-            lines.append(f'{osp.basename(dst_image_root)}/{dst_img_name} '
-                         f'{word}')
+            if format == 'txt':
+                lines.append(f'{osp.basename(dst_image_root)}/{dst_img_name} '
+                             f'{word}')
+            elif format == 'jsonl':
+                lines.append(
+                    json.dumps({
+                        'filename':
+                        f'{osp.basename(dst_image_root)}/{dst_img_name}',
+                        'text': word
+                    }))
```

Comment:
```suggestion
                    }， ensure_ascii=False))
```

`ensure_ascii=False` to support Chinese.